editor/src/components/canvas/controls/select-mode/canvas-strategy-picker.tsx

```diff
@@ -136,3 +136,33 @@ export const CanvasStrategyPicker = React.memo(() => {
     </>
   )
 })
+
+const KeyIndicator = ({ keyNumber }: { keyNumber: number }) => {
+  const height = 12
+  const width = 12
+  return (
+    <div
+      style={{
+        width,
+        height,
```

Comment:
we usually don't do mixed-style objects, some of us find it hard to read

```suggestion
        width: width,
        height: height,
```